packages/pigeon/platform_tests/test_plugin/ios/Classes/TestPlugin.swift

```diff
@@ -68,6 +59,39 @@ public class TestPlugin: NSObject, FlutterPlugin, HostIntegrationCoreApi {
     return aUint8List
   }
 
+  func extractNestedNullableString(wrapper: AllNullableTypesWrapper) -> String? {
+    return wrapper.values.aNullableString;
+  }
+
+  func createNestedNullableString(nullableString: String?) -> AllNullableTypesWrapper {
+    return AllNullableTypesWrapper(values: AllNullableTypes(aNullableString: nullableString))
+  }
+
+  func sendMultipleNullableTypes(aNullableBool: Bool?, aNullableInt: Int32?, aNullableString: String?) -> AllNullableTypes {
+    let someThings = AllNullableTypes(aNullableBool: aNullableBool, aNullableInt: aNullableInt, aNullableString: aNullableString)
+    return someThings
+  }
+
+  func echoNullableInt(aNullableInt: Int32?) -> Int32? {
+    return aNullableInt
+  }
+
+  func echoNullableDouble(aNullableDouble: Double?) -> Double? {
+    return aNullableDouble
+  }
+
+  func echoNullableBool(aNullableBool: Bool?) -> Bool? {
+    return aNullableBool
+  }
+
+  func echoNullableString(aNullableString: String?) -> String? {
+    return aNullableString
+  }
+
+  func echoNullableUint8List(aNullableUint8List: [UInt8]?) -> [UInt8]? {
```

Comment:
We probably don't want to add the Swift implementation for this one yet, since it's using the wrong types.

Reply:
PR to fix that is up now. I'm going to merge it with this one and update the tests once it's landed